### PR TITLE
Fzf localtime reltime pr

### PR DESCRIPTION
**Pull request by danihodovic**

### CODE REVIEW

File: autoload/fzf/vim.vim [hunk 1]
```diff
@@ -479,7 +479,8 @@ endfunction
 
 function! s:sort_buffers(...)
   let [b1, b2] = map(copy(a:000), 'get(g:fzf#vim#buffers, v:val, v:val)')
-  return b1 - b2
+  " Using minus between a float and a number in a sort function causes an error
+  return b1 > b2 ? 1 : -1
```
Comment (junegunn) [inline autoload/fzf/vim.vim]: Do we still need this hunk?

File: plugin/fzf.vim [hunk 2]
```diff
@@ -112,7 +112,7 @@ endif
 let g:fzf#vim#buffers = {}
 augroup fzf_buffers
   autocmd!
-  autocmd BufWinEnter,WinEnter * let g:fzf#vim#buffers[bufnr('')] = localtime()
+  autocmd BufWinEnter,WinEnter * let g:fzf#vim#buffers[bufnr('')] = reltimefloat(reltime())
```
Comment (junegunn) [inline plugin/fzf.vim]: A minor nit. There's a trailing space at the end.

Comment (danihodovic) [inline plugin/fzf.vim]: Fixed in c4786bd

**Comment by junegunn** [comment]
Squashed and merged, thanks.

**Comment by danihodovic** [comment]
You are quick!

**Pull request merged**